libraries/chain/webassembly/runtimes/eos-vm-oc/code_cache.cpp

```diff
@@ -306,6 +306,12 @@ code_cache_base::code_cache_base(const std::filesystem::path data_dir, const eos
 }
 
 void code_cache_base::set_on_disk_region_dirty(bool dirty) {
+   // tests can remove directory before destructor is called
```

Comment:
This feels pretty weird to me.. it makes it sound like there are cases where `fc::temp_dir` is dtor'ed before the `controller` is dtor'ed?

Reply:
Good point. I'll fix the tests and remove this.